apps/web/layouts/PageLayout/AccountMenu.tsx

```diff
@@ -0,0 +1,44 @@
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuTrigger,
+} from '@usevenice/ui'
+import {supabase} from '../../contexts/common-contexts'
+import {useSession} from '../../contexts/session-context'
+import type {ComponentPropsWithoutRef} from 'react'
+
+export function AccountMenu() {
+  const [session] = useSession()
+  return (
+    <DropdownMenu>
+      <DropdownMenuTrigger asChild>
+        <button
+          className="inline-flex focus:outline-none focus:ring-2 focus:ring-indigo-500 focus:ring-offset-2"
+          aria-label="Account menu">
+          {/* Somehow, we can't control size & fill with CSS. */}
+          <AccountCircleFilledIcon width={32} height={32} fill="currentColor" />
```

Comment:
Seems like fill only works when it's on the path for this scenario which is a bit odd. Setting `className="fill-green"` does nothing but `style={{fill: 'green'}}` seems to override it 🤷

Reply:
It's true that it'll be gone soon but we need to figure it out at some point since we'll run into the same issue when using SVG icon in other places.

In another project, I don't set anything on svg icons and can control size/fill on the wrapping element, something like:

```tsx
// inherits color from parents css `color`
<span className="inline-flex w-8 h-8 fill-current">
  <AccountCircleFilledIcon />
</span>
```